extensions/amp-ad-network-mytarget-impl/OWNERS

```diff
@@ -0,0 +1,14 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [
+        {name: 'ampproject/a4a'},
```

Comment:
Sorry, I wasn't clear here, as of https://github.com/ampproject/meta/issues/43 it seems we are deprecating the `a4a` group. Please remove it.

Reply:
Thank you. I've removed the deprecated option